docs/sources/auth/azuread.md

```diff
@@ -110,23 +115,28 @@ allowed_groups =
 
 > Note: Ensure that the [root_url]({{< relref "../administration/configuration/#root-url" >}}) in Grafana is set in your Azure Application Reply URLs (App -> Settings -> Reply URLs)
 
-### Configure allowed groups and domains
+### Configure allowed groups
 
-To limit access to authenticated users that are members of one or more groups, set `allowed_groups`
-to a comma- or space-separated list of group Object Ids. Object Id for a specific group can be found on the Azure portal: go to Azure Active Directory -> Groups. For instance, if you want to
-only give access to members of the group `example` which has Id `8bab1c86-8fba-33e5-2089-1d1c80ec267d`, set
+To limit access to authenticated users who are members of one or more groups, set `allowed_groups`
+to a comma- or space-separated list of group Object Ids. Object Ids for a specific group appear on the Azure portal:
+
+1. Go to Azure Active Directory -> Groups.
+
+For example, if you want to only give access to members of the group `example` which has Id `8bab1c86-8fba-33e5-2089-1d1c80ec267d`, set the following:
 
 ```ini
 allowed_groups = 8bab1c86-8fba-33e5-2089-1d1c80ec267d
 ```
 
-You'll need to ensure that you've [enabled group attributes](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-fed-group-claims#configure-the-azure-ad-application-registration-for-group-attributes) in your Azure AD Application Registration manifest file (Azure Portal -> Azure Active Directory -> Application Registrations -> Select Application -> Manifest)
+You'll need to ensure that you've [enabled group attributes](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-fed-group-claims#configure-the-azure-ad-application-registration-for-group-attributes) in your Azure AD Application Registration manifest file. Navigate to Azure Portal -> Azure Active Directory -> Application Registrations -> Select Application -> Manifest, and set the following:
```

Comment:
Per the [Documentation style guide](b/master/contribute/style-guides/documentation-style-guide.md) and [Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/welcome/), UI fields and things you click in the UI should be bold.